components/add_groups_to_channel_modal/add_groups_to_channel_modal.tsx

```diff
@@ -114,9 +132,9 @@ export default class AddGroupsToChannelModal extends React.PureComponent {
         });
     }
 
-    handleSubmit = async (e) => {
+    handleSubmit = async (e: Event | GroupValue[] | undefined) => {
```

Comment:
will this ever be undefined?

Reply:
I'm not sure but this is what the Multiselect component expects the handleSubmit function to be otherwise the compiler complains.

Reply:
regarding undefined: `Multiselect` component calls the function without parameters in some places, so it is needed.

I'm more concerned on why it might be an event or it might be a value, multiselect doesn't send the event as far as I could check, so why are we adding that to the type?